docs/en/index.rst

```diff
@@ -10,90 +10,83 @@
+   Processing-Request
+   Building-Response
+   Managing-Attributes
+   Calling-Alexa-Service-APIs
+   Configuring-Skill-Instance
 
-===================
+.. toctree::
+   :caption: REFERENCE
+   :hidden:
+
+   TypeDoc <http://ask-sdk-node-typedoc.s3-website-us-east-1.amazonaws.com/>
+
+*******************
 ASK SDK for Node.js
-===================
+*******************
 
-The ASK SDK v2 for Node.js makes it easier for you to build highly
-engaging skills by allowing you to spend more time implementing features
-and less time writing boilerplate code.
+The ASK SDK v2 for Node.js makes it easier for you to build highly engaging skills by allowing you to spend more time implementing features and less time writing boilerplate code.
 
 To help you get started with the SDK, see the following resources.
 
 Guides
-------
+======
 
 `Setting Up The ASK SDK`_
-~~~~~~~~~~~~~~~~~~~~~~~~~
+-------------------------
 
 Shows how to install the SDK as a dependency in your NPM project.
 
 `Developing Your First Skill`_
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+------------------------------
 
-Walks through step-by-step instructions for building the Hello World
-sample.
+Walks through step-by-step instructions for building the Hello World sample.
 
 `ASK SDK Migration Guide`_
-~~~~~~~~~~~~~~~~~~~~~~~~~~
+--------------------------
 
-Provides instructions for migrating an Alexa skill from the SDK v1 to
-the SDK v2.
+Provides instructions for migrating an Alexa skill from the SDK v1 to the SDK v2.
 
 Technical Documentation
------------------------
+=======================
 
-`Request Processing`_
-~~~~~~~~~~~~~~~~~~~~~
+`Processing Request`_
+---------------------
 
-Covers how to build request handlers, exception handlers, and request
-and response interceptors.
+Covers how to build request handlers, exception handlers, and request and response interceptors.
 
-`Skill Attributes`_
-~~~~~~~~~~~~~~~~~~~
+`Building Response`_
+--------------------
 
-Covers how to use skill attributes to store and retrieve skill data.
+Covers how to use the ResponseBuilder to compose multiple elements like text, cards, and audio into a single response.
 
-`Response Building`_
-~~~~~~~~~~~~~~~~~~~~
+`Managing Attributes`_
+----------------------
 
-Covers how to use the ResponseBuilder to compose multiple elements like
-text, cards, and audio into a single response.
+Covers how to use skill attributes to store and retrieve skill data.
 
-`Alexa Service Clients`_
-~~~~~~~~~~~~~~~~~~~~~~~~
+`Calling Alexa Service APIs`_
+-----------------------------
 
 Covers how to use service clients in your skill to access Alexa APIs.
 
-`Skill Builders`_
-~~~~~~~~~~~~~~~~~
+`Configuring Skill Instance`_
+-----------------------------
 
 Covers how to configure and construct a skill instance.
 
-`Managing In-Skill Purchases`_
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-Covers how to manage in-skill products and the purchase experience in
-your skills.
 
 Got Feedback?
--------------
+=============
```

Comment:
Can we add a section like

Other Language SDKs
=================

We also provide SDKs in Java and Python. Please have a look at the following resources : 

* [Java SDK](https://github.com/alexa/alexa-skills-kit-sdk-for-java)
* [Python SDK (Beta)](https://github.com/alexa-labs/alexa-skills-kit-sdk-for-python)

Reply:
I'm actually thinking about putting them in the TOC tree. Let's make sure we are consistent on this.

Reply:
Yeah indexing in the TOC tree is a better option. Let's do that as part of a separate PR for all 3 SDK's at once, outside of this PR.